# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libnetfilter_log/libnetfilter_log-0001-fix-includes.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

include sys/types.h for u_int*_t

--- ./src/libnetfilter_log.c.orig	2013-01-01 23:41:45.049308387 +0100
+++ ./src/libnetfilter_log.c	2013-01-01 23:42:21.014138091 +0100
@@ -25,6 +25,7 @@
 #include <time.h>
 #include <errno.h>
 #include <netinet/in.h>
+#include <sys/types.h>
 #include <sys/socket.h>
 
 #include <libnetfilter_log/linux_nfnetlink_log.h>
